src/test/java/com/spotify/futures/FuturesExtraTest.java

```diff
@@ -266,6 +267,68 @@ public void validate(Integer value) throws Exception {
   }
 
   @Test
+  public void testSuccessCallback() throws Exception {
+    final SettableFuture<Long> future = SettableFuture.create();
+    final SettableFuture<Long> value = SettableFuture.create();
+
+    FuturesExtra.addSuccessCallback(future, new FuturesExtra.Consumer<Long>() {
```

Comment:
I would prefer a Mockito.mock(Consumer.class) instead, and then use verify(mock).accept(10l)